test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -510,6 +510,19 @@ ConstructorTestClass(int i, bool b)
         $res.CompletionMatches[0].CompletionText | Should -BeExactly Cat
     }
 
+    it 'Should complete provider dynamic parameters with quoted path' {
+        $Script = if ($IsWindows)
+        {
+            'Get-ChildItem -Path "C:\" -Director'
+        }
+        else
+        {
+            'Get-ChildItem -Path "/" -Director'
+        }
```

Comment:
Is the test correct? I don't see the issue if I type first chars like `-D<tab>`.

Reply:
Yes the test is correct but it looks like you've found a different issue. Apparently it will tab complete it successfully with `-d` but if you type any more or any less it won't tab complete `-Directory` if the path is quoted.

Reply:
> Yes the test is correct but it looks like you've found a different issue.

Please clarify does the PR fix it?